docs/1-terms/D/term-design-principle.adoc

```diff
@@ -6,15 +6,15 @@
 Set of guidelines that helps software developers to design and implement better solutions,
 where "better" could, for example, mean one or more of the following:
 
-  * low <<term-coupling,coupling>>.
-  * high <<term-cohesion,cohesion>>.
-  * <<term-separation-of-concern,separation of concerns>> or adherence to the <<term-single-responsibility-principle,Single Responsibility Principle>>.
-  * adherence to the <<term-information-hiding,Information Hiding>> principle.
-  * avoid *Rigidity*: A system or element is difficult to change because every change potentially
-   affects many other elements.
-  * avoid *Fragility*: When elements are changed, unexpected results, defects or otherwise negative consequences
-   occur at other elements.
-  * avoid *Immobility*: An element is difficult to reuse because it cannot be disentangled from the rest of the system.
+* low <<term-coupling,coupling>>.
+* high <<term-cohesion,cohesion>>.
+* <<term-separation-of-concern,separation of concerns>> or adherence to the <<term-single-responsibility-principle,Single Responsibility Principle>>.
+* adherence to the <<term-information-hiding,Information Hiding>> principle.
+* avoid *Rigidity*: A system or element is difficult to change because every change potentially
+affects many other elements.
```

Comment:
should there really be a line break here?

Reply:
It's asciidoc, it doesn't matter.